src/applications/mhv-secure-messaging/util/threads.js

```diff
@@ -0,0 +1,9 @@
+export const getRecentMessages = (threadList, monthsAgo = 6) => {
```

Comment:
I would suggest to go with `getRecentThreads` as we are fetching threads, not messages, to keep nomenclature more precise